presto-main/src/main/java/com/facebook/presto/cost/JoinStatsRule.java

```diff
@@ -49,12 +49,21 @@
         implements ComposableStatsCalculator.Rule
 {
     private static final Pattern PATTERN = Pattern.typeOf(JoinNode.class);
+    private static final double DEFAULT_UNMATCHED_ANTI_JOIN_NDVS_COEFFICIENT = 0.5;
```

Comment:
did some initial benchmarks, there isn't regression (I think), current results q78:
```
ecj repart: 24s
ecj auto: 20s
cbo auto: 28s
```
Is it OK for now?